components/contact-details.tsx

```diff
@@ -41,15 +41,17 @@ const DescriptionItem = (props: DescriptionItemProps) => {
       <dd className="mt-1 flex text-sm text-gray-900 sm:mt-0 sm:col-span-2">
         <span className="flex-grow">
           {htmr(value, { transform: htmrTransform })}
+        </span>
+        <div className="flex flex-wrap justify-end space-y-1 content-start">
```

Comment:
```suggestion
        <div className="flex flex-wrap flex-none justify-end space-y-1 content-start">
```

Add a `flex-none` so the size of these buttons column won't auto-adjust and cause this issue.

![image](https://user-images.githubusercontent.com/5663877/127737480-9a310bcb-89a8-4f2a-9dd3-6bb0e74c0343.png)

Reply:
Do you mind elaborating more on the issue? Do you want to have the `Buka Peta` text in the same line instead of wrapping? Adding `flex-none` will make the buttons in the same row, as in the following image.
![Screenshot 2021-07-31 at 6 55 02 PM](https://user-images.githubusercontent.com/35729243/127737719-02c204c7-ffaf-41b0-a667-42ac536d1cf6.png)

Reply:
Or is this what you are looking for?
![Screenshot 2021-07-31 at 7 23 15 PM](https://user-images.githubusercontent.com/35729243/127738363-54630a73-7d62-4fa4-bd46-0a09f2b6bcb0.png)

Reply:
Yes, like that.

Reply:
Here is the latest state with 9ed82ca changes.
![Screenshot 2021-08-01 at 1 08 01 PM](https://user-images.githubusercontent.com/35729243/127759937-ceec211f-178b-4e1d-a462-f6d0d5940560.png)
![Screenshot 2021-08-01 at 1 07 44 PM](https://user-images.githubusercontent.com/35729243/127759938-da25548c-cf02-42f4-b2e4-ae69d3ea1d81.png)